app/viewmodels/IndexPageHelper.scala

```diff
@@ -228,4 +236,19 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
     }
     else None
   }
+
+  private def callObligationAPI(vrn: String)(implicit ec: ExecutionContext, hc: HeaderCarrier, user: User[_]) = {
+    complianceService.getDESComplianceData(vrn).map {
+      dataOpt => {
+        Try(dataOpt.get).fold(
```

Comment:
This Try is a bit redundant, you could just to _.fold and that would do the same thing